app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="bg-dark-blue">
+  <h2 class="text-center p-4 text-light
+">Log in</h2>
+</div>
+<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="w-50 mx-auto m-4">
```

Comment:
Same thing with logging in and signing up. Kindly make sure the input fields are nearly 100% of the mobile view width as per design specifications.

![image](https://user-images.githubusercontent.com/38088818/215268443-16c22be8-b8d3-4c86-bc05-ee30057ec695.png)

![image](https://user-images.githubusercontent.com/38088818/215268473-e8fafd42-33a7-4633-aa77-30cd8195f54c.png)


- Otherwise, you have made a great effort in improving your design. Great job!!! 💯